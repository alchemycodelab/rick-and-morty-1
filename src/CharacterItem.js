import React, { Component } from 'react'

export default class CharacterItem extends Component {
    render() {
        return (
           
               <li className='characterItem'>
                   <h2 className='characterName'>{this.props.obj.character}</h2>
                   <img className='characterImage' src={this.props.obj.image} alt={this.props.obj.character}/>
                   <p className='species'>{this.props.obj.species}</p>
                   <p className='status'>{this.props.obj.status}</p>
               </li>
           
        )
    }
}
