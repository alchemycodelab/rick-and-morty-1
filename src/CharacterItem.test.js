import React from 'react';
import { shallow } from 'enzyme';
import CharacterItem from './CharacterItem';

it('renders welcome message', () => {
    const wrapper = shallow(<CharacterItem obj={{ character: 'some character', 
    image: 'some image', species: 'some species', status: 'some status'}}/>);
  
    expect(wrapper).toMatchSnapshot();
  });