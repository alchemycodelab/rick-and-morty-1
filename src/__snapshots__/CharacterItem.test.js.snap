// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome message 1`] = `
<li
  className="characterItem"
>
  <h2
    className="characterName"
  >
    some character
  </h2>
  <img
    alt="some character"
    className="characterImage"
    src="some image"
  />
  <p
    className="species"
  >
    some species
  </p>
  <p
    className="status"
  >
    some status
  </p>
</li>
`;
